### ISSUE TITLE
Typo in the doc for load

### ISSUE THREAD

**Issue by dmathieu** [eee-c/couch_docs#4]
Hello,

In the README file, you're saying :

```
couch-docs load "http://localhost:5984/db" path/to/dump_dir/
```

However the appropriate command is the following :

```
couch-docs load path/to/dump_dir/ "http://localhost:5984/db"
```

**Comment by eee-c** [comment]
Actually load is deprecated, it should be 
    couch-docs push "http://localhost:5984/db" path/to/dump_dir/

I made the change to be more in-line with couchapp and to make the directory optional (defaults to the current directory).

I will update the README accordingly.  Thanks!

**Comment by eee-c** [comment]
Update the README to reflect preference for "push" rather than "load". closed by 713087bfd3378032b0eded91059b842677a6ac53
